definition-map.js

```diff
@@ -90,6 +90,50 @@ const QUERIES = {
             quizzes: "The IDs of the Quizzes user selects."
         }
     },
+    Forums: {
+        metadescription: "This query returns a forum containing discussion threads within a course, discussion widget, or assignment.",
+        args: {
+            courseId: "The ID of the course.", 
```

Comment:
`course`/`client` seem to vary in capitalization through the descriptions, so will want to make those consistent.